apps/src/aichat/views/ModelCustomizationWorkspace.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import Tabs from './tabs/Tabs';
+
+const ModelCustomizationWorkspace: React.FunctionComponent = () => {
+  return (
+    <div style={{width: 400}}>
```

Comment:
nit: what do you think of using `aichatView.module.scss` as we do for `ChatWorkspace` at https://github.com/code-dot-org/code-dot-org/blob/53b937168ad0533fbecebe73d27f6964cc438141/apps/src/aichat/views/AichatView.tsx#L41

Reply:
Sounds like a good idea to manage the width in AichatView! I have a lot of CSS changes in the follow-up PR adding content to the tab panels, so I just added [a commit there](https://github.com/code-dot-org/code-dot-org/pull/57095/commits/33906c50a74db44548135fe00fe6c9a65d467420).